apps/application-system/form/src/styles.scss

```diff
@@ -0,0 +1,45 @@
+@font-face {
```

Comment:
Should these be part of core theme?

Reply:
probably, I just don't know how to do it in another way :/ just followed Kolibri's lead on this.

Reply:
Agree, it should probably be moved to the island-ui. One scss file that could be imported along with the fonts folder